core/math/audio_frame.h

```diff
@@ -51,105 +51,123 @@ static const float AUDIO_PEAK_OFFSET = 0.0000000001f;
+	_ALWAYS_INLINE_ AudioFrame operator*(const AudioFrame &p_frame) const { return AudioFrame(left * p_frame.left, right * p_frame.right); }
+	_ALWAYS_INLINE_ AudioFrame operator/(const AudioFrame &p_frame) const { return AudioFrame(left / p_frame.left, right / p_frame.right); }
 
-	_ALWAYS_INLINE_ AudioFrame operator+(float p_sample) const { return AudioFrame(l + p_sample, r + p_sample); }
-	_ALWAYS_INLINE_ AudioFrame operator-(float p_sample) const { return AudioFrame(l - p_sample, r - p_sample); }
-	_ALWAYS_INLINE_ AudioFrame operator*(float p_sample) const { return AudioFrame(l * p_sample, r * p_sample); }
-	_ALWAYS_INLINE_ AudioFrame operator/(float p_sample) const { return AudioFrame(l / p_sample, r / p_sample); }
+	_ALWAYS_INLINE_ AudioFrame operator+(float p_sample) const { return AudioFrame(left + p_sample, right + p_sample); }
+	_ALWAYS_INLINE_ AudioFrame operator-(float p_sample) const { return AudioFrame(left - p_sample, right - p_sample); }
+	_ALWAYS_INLINE_ AudioFrame operator*(float p_sample) const { return AudioFrame(left * p_sample, right * p_sample); }
+	_ALWAYS_INLINE_ AudioFrame operator/(float p_sample) const { return AudioFrame(left / p_sample, right / p_sample); }
 
 	_ALWAYS_INLINE_ void operator+=(const AudioFrame &p_frame) {
-		l += p_frame.l;
-		r += p_frame.r;
+		left += p_frame.left;
+		right += p_frame.right;
 	}
 	_ALWAYS_INLINE_ void operator-=(const AudioFrame &p_frame) {
-		l -= p_frame.l;
-		r -= p_frame.r;
+		left -= p_frame.left;
+		right -= p_frame.right;
 	}
 	_ALWAYS_INLINE_ void operator*=(const AudioFrame &p_frame) {
-		l *= p_frame.l;
-		r *= p_frame.r;
+		left *= p_frame.left;
+		right *= p_frame.right;
 	}
 	_ALWAYS_INLINE_ void operator/=(const AudioFrame &p_frame) {
-		l /= p_frame.l;
-		r /= p_frame.r;
+		left /= p_frame.left;
+		right /= p_frame.right;
 	}
 
 	_ALWAYS_INLINE_ void operator+=(float p_sample) {
-		l += p_sample;
-		r += p_sample;
+		left += p_sample;
+		right += p_sample;
 	}
 	_ALWAYS_INLINE_ void operator-=(float p_sample) {
-		l -= p_sample;
-		r -= p_sample;
+		left -= p_sample;
+		right -= p_sample;
 	}
 	_ALWAYS_INLINE_ void operator*=(float p_sample) {
-		l *= p_sample;
-		r *= p_sample;
+		left *= p_sample;
+		right *= p_sample;
 	}
 	_ALWAYS_INLINE_ void operator/=(float p_sample) {
-		l /= p_sample;
-		r /= p_sample;
+		left /= p_sample;
+		right /= p_sample;
 	}
 
 	_ALWAYS_INLINE_ void undenormalize() {
-		l = ::undenormalize(l);
-		r = ::undenormalize(r);
+		left = ::undenormalize(left);
+		right = ::undenormalize(right);
 	}
 
 	_FORCE_INLINE_ AudioFrame lerp(const AudioFrame &p_b, float p_t) const {
 		AudioFrame res = *this;
 
-		res.l += (p_t * (p_b.l - l));
-		res.r += (p_t * (p_b.r - r));
+		res.left += (p_t * (p_b.left - left));
+		res.right += (p_t * (p_b.right - right));
 
 		return res;
 	}
 
 	_ALWAYS_INLINE_ AudioFrame(float p_l, float p_r) {
-		l = p_l;
-		r = p_r;
+		left = p_l;
+		right = p_r;
```

Comment:
Not entirely needed but the parameters could be renamed here too.

Reply:
Avoided it to reduce the impact but can do it there too